src/Message/PushNotificationV2.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace AppBundle\Message;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class PushNotificationV2
```

Comment:
I don't understand why this class is necessary?
Wasn't it more simple to add a `title` property to the existing class, and use the title if not empty when creating the notification?
Seems overkill to have 2 classes, just to support notification titles.

Reply:
Done what @vladimir-8 suggested at: https://github.com/coopcycle/coopcycle-web/pull/5062/commits/35eef1790809a71cec743df8b5c956942d726a09